simple_safer_server/routes/storage.py

```
@@ -135,24 +256,36 @@ def shutdown():
-            used_storage = total_storage = storage_usage = None
-    else:
+    storage_error = status["error"]
+    try:
+        # Usage only needs the path to be readable. The stricter available/error
+        # fields avoid marker/probe I/O so opening the dashboard does not wake
+        # drives just to prove cloud-backup safety.
+        disk = psutil.disk_usage(mount_point)
+        used_storage = f"{disk.used / (1024**3):.1f}"
+        total_storage = f"{disk.total / (1024**3):.1f}"
+        storage_usage = f"{disk.percent}%"
+    except Exception:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)